src/components/HomePage.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable camelcase */
+// the API data dosent use camel case!
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+
+const dateFormater = () => {
+  const date = new Date();
+
+  const d = date.getDate();
+  let m = date.getMonth();
+  const y = date.getFullYear();
+  m += 1;
+
+  const dateString = `${y}-${m <= 9 ? `0${m}` : m}-${(d <= 9 ? `0${d}` : d)}`;
+  return dateString;
+};
+
+export function Country({ name, today_confirmed }) {
+  return (
+    <div className="card">
+      <NavLink to={`/${name}`}>
+        <li className="countryCard">
+          <div className="mapBox">
+            <img
+              className="map"
+              src={`https://mapsvg.com/static/maps/geo-calibrated/${name.toLowerCase()}.svg`}
+              alt="map"
+              /* eslint-disable no-param-reassign */
+              onError={(event) => {
+                event.target.src = 'https://img.icons8.com/pastel-glyph/120/000000/map.png';
+                event.onerror = null;
+              }}
+            />
+          </div>
+          <h4>{name}</h4>
+          <h4>{today_confirmed}</h4>
+        </li>
+      </NavLink>
+    </div>
+  );
+}
+
+Country.propTypes = {
+  name: PropTypes.string.isRequired,
+  today_confirmed: PropTypes.number.isRequired,
+};
+
```

Comment:
- [ ] Optional suggestion:

- Same thing goes here too,  I would recommend including/adding it in the below lines after a return to make our code look clean and clear